### PR TITLE
Fixed an anchor and corrected a typo.

### PR DESCRIPTION
**Pull request by neotan**
Updated in index.bs file, instead of index.html

### CODE REVIEW

**Comment by marcoscaceres** [comment]
You are a hero, @neotan. Sorry you had to do extra work!

**Comment by neotan** [comment]
Never mind, @marcoscaceres. And thanks @miguelao for your straightforward guidance of  **W3C CSS Spec Preprocessor**, it really help me out.

Btw, I am considering to introduce this spec to Chinese developer, i.e. translating into Chinese and provide a `index-zh-cn.bs`, do you guys think it's proper?

**Comment by neotan** [comment]
I just downloaded Chrome Canary(Version 58.0.3021.1 canary SyzyASan) and enabled `chrome://flags/#enable-experimental-web-platform-features` flag already. But [sample1](https://output.jsbin.com/gegudoc/4/) and [sample2](https://codepen.io/miguelao/pen/ZBBxOM?editors=0010) don't work now. 

May I know is there any change at detection API level? Any code need to be adjust? Many thanks.

**Comment by miguelao** [comment]
@neotan, re. 
- the first example: s/faces[i].x/faces[i].boundingBox.x/ etc, that'll do the trick
- the second example: oblique QR codes do not work well in Mac, try in Android, or another easier case such as, e.g. [this one](https://upload.wikimedia.org/wikipedia/en/f/f4/LiveBarcodeApp.jpg) ?
- the chinese translation: absolutely! I wonder how can we make evident in the Spec that you're the one in charge of such translation (like having a specific entry in the `.bs`), and how to keep it in sync with the english one, @marcoscaceres , any savvy tips here?

**Comment by neotan** [comment]
Hi @miguelao and @marcoscaceres , I just committed a Chinese version (dbceb8a). Kindly pls have a check is there anything I can improve. Many thanks.

**Comment by marcoscaceres** [comment]
@neotan, it looks good to me (though sadly I don't  read Chinese).

**Comment by neotan** [comment]
@marcoscaceres ,hah~ believe me, I spent few hours to make this Chinese be perfect.  

I'm already push the update to the repo of my fork, is it good to merge/accpet at your side now? Sorry, I am newbee to git.

**Comment by marcoscaceres** [comment]
@neotan, as I'm not the Editor of the spec, I will leave it for @miguelao to merge.

**Comment by neotan** [comment]
Understood @marcoscaceres . thx.

Btw, I am writing a news for introducing the these features to Chinese developers, want to attach this spec's english and chinese version to the news as references. 

Do you guys think is it good? I mean, may I have your consent? @miguelao

File: index.bs [hunk 1]
```diff
@@ -99,7 +99,7 @@ interface FaceDetector {
   <dt><dfn constructor for="FaceDetector"><code>FaceDetector(optional FaceDetectorOptions faceDetectorOptions)</code></dfn></dt>
   <dd>Constructs a new {{FaceDetector}} with the optional |faceDetectorOptions|.</dd>
```
Comment (miguelao) [inline index.bs]: Please add to the index.bs `metadata` an entry `Translation` (search for `Translation` in https://github.com/mikewest/bikeshed/blob/master/docs/metadata.md).

Actually, WDYT if we 
a) land the modifications to `index.bs`  (and `index.html`) now, in this PR
b) move the addition of the chinese translation to _another_ PR, which will include adding the `Translation` entry in `index.bs` (and there should be another correspondent `Translation` entry in index-zh-cn.bs` indicating the english version).
That way it'll be easier to track the history of changes.

**Comment by miguelao** [comment]
Oh, re. 
> Do you guys think is it good? I mean, may I have your consent? @miguelao

By all means go ahead :smile:: whatever is in this repo is of public access.

File: index.bs [hunk 2]
```diff
@@ -129,7 +129,7 @@ Example implementations of face detection are e.g. <a href="https://developer.an
...
+{{BarcodeDetector}} represents an underlying accelerated platform's component for detection in images of QR codes or barcodes.  It provides a single {{BarcodeDetector/detect()}} operation on an {{ImageBitmapSource}} which result is a Promise.  This method MUST reject this Promise in the cases detailed in [[#image-sources-for-detection]]; otherwise it MAY queue a task using the OS/Platform resources to resolve the Promise with a sequence of {{DetectedBarcode}}s, each one essentially consisting on and delimited by a {{DetectedBarcode/boundingBox}} and a series or {{Point2D}}s, and possibly a {{DetectedBarcode/rawValue}} decoded {{DOMString}}.
```
Comment (miguelao) [inline index.bs]: Remove the changes to index.bs and index.html from this PR, since they'll land as part of the #21.

File: index-zh-cn.bs [hunk 3]
```diff
@@ -0,0 +1,355 @@
+<pre class="metadata">
+Title: Accelerated Shape Detection in Images
+Repository: wicg/shape-detection-api
+Status: ED
+ED: https://wicg.github.io/shape-detection-api
+Shortname: shape-detection-api
+Level: 1
+Editor: Miguel Casas-Sanchez, w3cid 82825, Google Inc., mcasas@google.com
+Abstract: 本文档描述了一套Chrome中针对静态和/或动态图像的图形识别（如：人脸识别）API。
```
Comment (miguelao) [inline index-zh-cn.bs]: Add here a line:
`Translation: en-US https://wicg.github.io/shape-detection-api`

and to the index.bs, add a line:
`Translation: zh-CN https://wicg.github.io/shape-detection-api/index-zh-cn.html`

And don't forget compiling both by using the appropriate bikeshed service, e.g.
`curl https://api.csswg.org/bikeshed/ -F file=@index.bs > index.html`

Comment (neotan) [inline index-zh-cn.bs]: Ok, done them.

**Comment by neotan** [comment]
The news is posted at infoQ.com(China) now: http://www.infoq.com/cn/news/2017/03/chrome-shape-detection
Thank you all!

**Comment by marcoscaceres** [comment]
Thank you for helping push the spec forward and getting the word out!

**Comment by marcoscaceres** [comment]
@neotan getting a 404 on that link, btw

**Comment by miguelao** [comment]
I got a 404 as well.

**Comment by neotan** [comment]
Opps~ the website just adjusted the URL: http://www.infoq.com/cn/news/2017/03/Chrome-Shape-Detection-API

**Pull request merged**